templates/TOPStyleGuide.md

```diff
@@ -24,131 +25,146 @@ TOP uses Markdown for its lessons to be easily read and edited, and get properly
     1.  [Codeblocks](#codeblocks)
     1.  [Declare the language](#declare-the-language)
     1.  [Nest codeblocks within lists](#nest-codeblocks-within-lists)
+1.  [Links](#links)
+1.  [Images](#images)
 
 
-## Lesson Layout
+## Layouts
+
+In general, the following layouts should be used for all lessons and projects.
+
+### Lesson Layout
 
 ~~~markdown
 
 ### Introduction
 
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+A brief introduction.
 
 ### Learning Outcomes
 
-*Look through these now and then use them to test yourself after doing the assignment*
+This section contains a general overview of topics that you will learn in this lesson.
+
+*   A learning outcome.
 
-what the student is expected to know or be able to do by the end of this lesson
+### Custom Section Heading
 
-*   Learning outcome 1
-*   Learning outcome 2
-*   Learning outcome 3
+Custom section content.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-The list of resources the user will go through to learn about the topic of this lesson. Have no more than 5 resources, ideally no more than three.
+1. A resource/task.
 </div>
 
-### Exercises
+### Knowledge Check
+
+This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
 
-A group of exercises (If Applicable) for the student to complete in relation to the topic taught in the lesson. 
+*   <a class="knowledge-check-link" href="A knowledge check URL">A knowledge check question.</a>
```

Comment:
Could the href example mirror a markdown header link and maybe use all caps to imply it needs to be replaced? Just a thought. `href="#REPLACE-ME"` or `href="#KNOWLEDGE-CHECK-LINK"` .This would make it easier when copying and pasting the template when writing a lesson because it "jumps out" when it's not replaced.

Reply:
Not using either type of link as the default text for the `href` attribute was purposeful, but I do see your point. Unlike the rest of the default text used in the template, the KC instance does have that default text kind of crammed in with other text which I could see being easy to gloss over.

Won't make any change jus tyet in case anyone else has any thoughts, but I'm gonna keep this in mind.

Reply:
I ended up going for more of a middle-ground approach. I wanted to refrain from defaulting to any type of link for the KC's, but I do agree it was difficult to pick out what text should be edited within that chunk of text.

I opted to update both layout files so that any text that *should* be replaced by the author is in all caps throughout.